include/lwipopts.h

```diff
@@ -74,6 +74,9 @@ a lot of data that needs to be copied, this should be set high. */
    no copy is needed */
 #define LWIP_SUPPORT_CUSTOM_PBUF 1
 
+/* --------- Set hostname of device on network ---------- */
+#define LWIP_NETIF_HOSTNAME 1
```

Comment:
Im not actually sure what this does? Can we remove?

Reply:
Setting this back to zero as i'm having networking issues to get this working.